.eslintrc.js

```diff
@@ -14,9 +14,10 @@ module.exports = {
     // node files
     {
       files: [
+        'ember-cli-build.js',
         'index.js',
         'testem.js',
-        'ember-cli-build.js',
+        'blueprints/*/index.js',
```

Comment:
maybe change this to "blueprings/**/*.js"? 

Then we don't need `/* eslint-env node */` in the [other](https://github.com/typed-ember/ember-cli-typescript/pull/210/files#diff-30ab6740525727ef1fcb03b1eb97747cR1) node files inside the "blueprints/" dir.